lib/laa_crime_schemas/structs/crime_application.rb

```diff
@@ -29,6 +29,8 @@ class CrimeApplication < BaseApplication
       attribute? :work_stream, Types::WorkStreamType.optional
 
       attribute? :additional_information, Types::String.optional
+
+      attribute? :has_partner, Types::String.optional
```

Comment:
Need to rename `client_has_partner` to `has_partner` on apply

Reply:
Are we being consistent with there we store 'has_partner'? Looking at the :relationship_to_partner, :separation_date, :relationship_status which is stored on Applicant in this PR

Reply:
Good point, will move to the applicant